File: C++/Codechef - GCD and LCM.cpp
#include <bits/stdc++.h>
using namespace std;

int gcd(int a, int b){
    return b?gcd(b, a%b):a;
}

int lcm(int a, int b){
    return a/gcd(a,b)*b;
}

int main() {
	int n;cin>>n;
	
	for (int i=0;i<n;++i){
        int a,b;
        cin>>a>>b;
        
        int res_gcd=gcd(a,b);
        int res_lcm=lcm(a,b);	
	    cout<<res_gcd<<" "<<res_lcm<<"\n";
	}
	return 0;
}

File: Python/Codechef - GCD and LCM.py
def gcd(a, b):
    if b==0: # base case
        return a
    return gcd(b, a%b)

def lcm(a, b):
    if b==0:
        return a
    return a/gcd(b, a%b)*b

def main(n):
    for _ in range(n):
        a,b = map(int, input().split())
        print(gcd(a,b), lcm(a,b))
main(int(input()))
